app/views/catalog/_display_order_manager_button.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if can?(:order_manager, resource) %>
+  <% if decorated_resource.orderable_files? && !decorated_resource.is_a?(NumismaticMonogramDecorator) %>
```

Comment:
I think it would be nice to create a new method on ChangeSet called something like `manageable_order?` to control whether this button is displayed or not.

Reply:
ok will do, thanks